python/pyspark/sql/tests/pandas/test_pandas_transform_with_state.py

```diff
@@ -26,13 +26,7 @@
 
 from pyspark import SparkConf
 from pyspark.sql.functions import split
-from pyspark.sql.types import (
-    StringType,
-    StructType,
-    StructField,
-    Row,
-    IntegerType,
-)
+from pyspark.sql.types import StringType, StructType, StructField, Row, IntegerType, TimestampType
```

Comment:
nit: probably linter may complain?

Reply:
Ran the linter check and seems not complaining... If it is OK with you, I'll leave it here and see if it passes on the CI.